src/components/Card/WCard.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div v-if="isLinkCard">
+    <a
+      :href="url"
+      :aria-label="title"
+      target="_blank"
+      class="cursor-pointer"
+      @mouseenter="toggleHover"
+      @mouseleave="toggleHover"
+    >
+      <div
+        class="card border rounded-lg px-xl py-xl mr-sm mb-sm max-w-md h-full"
+      >
+        <div class="flex justify-between items-center">
+          <h3 class="text-xl text-black-950 font-semibold leading-4">
+            {{ title }}
+          </h3>
+          <AnimatedArrow :hover="hover" color="#C6C6C6" />
+        </div>
+        <hr class="hr my-md" />
+        <slot name="description" class="text-sm text-black-950"></slot>
+      </div>
+    </a>
+  </div>
+  <div
+    v-else class="h-full w-auto out-of-boundaries card border-2 border-black-950 bg-white-50 rounded-lg px-xl py-xl max-w-sm"
+  >
+  <div v-if="isBaseCard">
```

Comment:
I think the most basic  card should contain a `header` and `content` slot splitted by a divider. We don't have a card without them.

![image](https://github.com/user-attachments/assets/07b84fa3-de1a-4f10-878e-59d95b42ae6b)